### PR TITLE
openrgb

### PR DESCRIPTION
**Pull request by Macleykun**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95650)

### CODE REVIEW

File: manifests/c/CalcProgrammer1/OpenRGB/0.8/CalcProgrammer1.OpenRGB.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: CalcProgrammer1.OpenRGB
+PackageVersion: "0.8"
+InstallerLocale: en-US
+InstallerType: zip
+ReleaseDate: 2022-11-28
+Installers:
+- Architecture: x64
```
Comment (Trenly) [inline manifests/c/CalcProgrammer1/OpenRGB/0.8/CalcProgrammer1.OpenRGB.installer.yaml:10]: Please add the x86 version

**Comment by Trenly** [comment]
This has a dependency on the VC Runtime too
image

**Comment by Macleykun** [comment]
> This has a dependency on the VC Runtime too
> image

Is it possible to add this dependency in the manifest?

**Comment by Macleykun** [comment]
> This has a dependency on the VC Runtime too image

If i edit the PR to make the manifest like so:
```
# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json

PackageIdentifier: CalcProgrammer1.OpenRGB
PackageVersion: "0.8"
InstallerLocale: en-US
InstallerType: zip
ReleaseDate: 2022-11-28
Installers:
- Architecture: x86
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: './OpenRGB Windows 64-bit/OpenRGB.exe'
    PortableCommandAlias: openrgb
  InstallerUrl: https://openrgb.org/releases/release_0.8/OpenRGB_0.8_Windows_64_fb88964.zip
  InstallerSha256: 8715A4B2E8CC821A53C861B16EE5AE00AC8BBED10009A6CABFC538367A1EA030
ManifestType: installer
ManifestVersion: 1.4.0
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```
Would that work? (after the feature is implemented, released and has a high enough adoption)?
I would say the latest version is always good.

**Comment by Macleykun** [comment]
Would the changes i've made, but sufficient that when manifest v1.5 comes out and is rolled out to most windows users. That it can be merged?

**Comment by Macleykun** [comment]
Heya, i tested it out and it worked in preview.
But i do wonder, will it also create a shortcut somehow in the future? B/c it's now quite hard to reach/find the .exe file to start openrgb.exe.
Sorry if i'm asking in the wrong place

**Comment by Trenly** [comment]
> Heya, i tested it out and it worked in preview. But i do wonder, will it also create a shortcut somehow in the future? B/c it's now quite hard to reach/find the .exe file to start openrgb.exe. Sorry if i'm asking in the wrong place

Currently there aren't any provisions to add shortcuts for portable applications. Once Dependencies are stable, then the PR will be re-run against the pipelines

**Comment by Macleykun** [comment]
hello!
I just installed vmware workstation pro and noticed it installed dependencies
Double checking here: https://github.com/microsoft/winget-pkgs/blob/master/manifests/v/VMware/WorkstationPro/17.0.2/VMware.WorkstationPro.installer.yaml i assume dependency support is stable?

**Comment by denelon** [comment]
Dependency support is still experimental. We're going through lots of edge cases, and several manifests were allowed ahead of the policy change, so we "grandfathered" them in. On occasion, we will allow other packages with dependencies in, but that's primarily so we can get data on all the permutations we need to be able to support.

We're getting close to the point where this could be stable and allow all the packages with "same source" dependencies in, but we're not quite there yet.

I think the next "big" improvement needed is the reboot resilience work.
* https://github.com/microsoft/winget-cli/issues/3165

We have several issues with the [Dependencies](https://github.com/microsoft/winget-cli/issues?q=is%3Aissue+is%3Aopen+label%3ADependencies) label to get a more complete picture.

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

File: manifests/c/CalcProgrammer1/OpenRGB/0.8/CalcProgrammer1.OpenRGB.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: CalcProgrammer1.OpenRGB
+PackageVersion: "0.8"
+InstallerLocale: en-US
+InstallerType: zip
+ReleaseDate: 2022-11-28
+Installers:
```
Comment (stephengillie) [inline manifests/c/CalcProgrammer1/OpenRGB/0.8/CalcProgrammer1.OpenRGB.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
Installers:
```

**Comment by Macleykun** [comment]
Thanks for the review!

**Comment by denelon** [comment]
We're going to go ahead and make dependency support stable in WinGet 1.6 without:
* #3165

It seems to be a small enough edge case that it doesn't seem to justify blocking the feature. We will continue working on the reboot resilience and hope to get that out in the next release or two.

**Comment by Macleykun** [comment]
> We're going to go ahead and make dependency support stable in WinGet 1.6 without:
> 
> * [Add Rust 1.45.2 MSVC x64 #3165](https://github.com/microsoft/winget-pkgs/pull/3165)
> 
> It seems to be a small enough edge case that it doesn't seem to justify blocking the feature. We will continue working on the reboot resilience and hope to get that out in the next release or two.

Awsome! So that means this can be merged?

**Comment by Macleykun** [comment]
Heyo
I was only wondering if this PR can be merged or are there any features/time frames we have to wait for?

(Didn’t meant to ask for a faster review)

**Comment by stephengillie** [comment]
Hi @Macleykun,

Yes - we just had to have eyes on it. Feel free to @ me in the future.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**